hbase-client/src/main/java/org/apache/hadoop/hbase/ipc/DefaultNettyEventLoopConfig.java

```diff
@@ -31,10 +30,16 @@
  */
 @InterfaceAudience.Private
 class DefaultNettyEventLoopConfig {
```

Comment:
This is not the design...

The design is to create a new EventLoop and make the code use it.

Change a configuration to change the 'default' instance and then make the code use the 'default' instance seems strange...

And also another anti pattern is that, we even do not have a Configuration instance here...

Reply:
Or let's just remove this class. Move the creation of event loop into the NettyRpcClientConfigHelper directly and honor the hbase.netty.worker.count configuration when creating? Though a bit strange is that the Configuration passed in may change but at least it can avoid reading the configuration from system property...